dev/tools/localization/bin/gen_l10n.dart

```diff
@@ -72,6 +72,21 @@ void main(List<String> arguments) {
       'Alternatively, see the `header` option to pass in a string '
       'for a simpler header.'
   );
+  parser.addFlag(
+    'use-deferred-loading',
+    defaultsTo: false,
+    help: 'Whether to generate the Dart localization file with locales imported'
+      ' as deferred, to allow for lazy loading of localizations in Flutter web.'
+      '\n\nThis can reduce a web app’s initial startup time by decreasing the '
```

Comment:
I also try to keep the string in this file neat to help with readability: 
```suggestion
      ' as deferred, to allow for lazy loading of localizations in Flutter web.\n\n'
      'This can reduce a web app’s initial startup time by decreasing the '
```